src/item.rs

```diff
@@ -75,7 +75,6 @@ struct LevelVisitor;
 impl<'de> Visitor<'de> for LevelVisitor {
     type Value = Levels;
 
-    #[cfg(not(tarpaulin_include))]
```

Comment:
there is no use of tarpaulin in this project.
And now coverage is best done with tools other than tarpaulin